config-templates/redhat/odf-local/changelog.json

```diff
@@ -3,6 +3,15 @@
         "4.12": {
             "status": "supported",
             "latest-revision": {
+                "revision": "2",
+                "release-date": "04 May 2023",
+                "cves": "CVE-2023-0361, CVE-2023-24536, CVE-2023-24537, CVE-2023-24538",
+                "golang": "1.19.8",
+                "ubi": "8.7-1107",
+                "new-features": [],
+                "fixes": ["LSO channel changed"]
```

Comment:
If applicable, can you please add more detail about `LSO channel change`?
Like what it was earlier and whats difference now in new revision?